config/settings/base.py

```diff
@@ -156,4 +156,4 @@
 # CURRENT KATSU VERSION ACCORDING TO MODEL CHANGES
 # ------------------------------------------------
 
-KATSU_VERSION = "3.0.0"
+KATSU_VERSION = "3.1.0"
```

Comment:
Are date_intervals a breaking change? If so, we should bump it by a whole number.

Reply:
sure